c2corg_ui/templates/helpers/view.html

```diff
@@ -397,11 +397,11 @@ <h3 class="heading closed show-phone" ng-click="mainCtrl.animateHeaderIcon($even
 
 
 ## Float buttons
-<%def name="show_float_buttons(outingUsers, doctype, doc_id, doc_lang, other_langs, missing_langs)">\
+<%def name="show_float_buttons(associatedUsers, doctype, doc_id, doc_lang, other_langs, missing_langs, image_type = 'default')">\
   <div class="float-buttons">
-      <div ng-if="::userCtrl.hasEditRights(${dumps(outingUsers)})" class="float-button float-edit"
-           tooltip-placement="left" uib-tooltip="{{'Edit' | translate}}"
-           protected-url-btn url="${request.route_path(doctype + '_edit', id=doc_id, lang=doc_lang)}">
+      <div ng-if="'${image_type}' == 'collaborative' || userCtrl.hasEditRights(${dumps(associatedUsers)})" class="float-button float-edit"
+       tooltip-placement="left" uib-tooltip="{{'Edit' | translate}}"
+       protected-url-btn url="${request.route_url(doctype + '_edit', id=doc_id, lang=doc_lang)}">
```

Comment:
Are you sure of this ``request.route_url``? It looks like a regression after @tsauerwein has changed it to ``request.route_path``.